bundlewrap/utils/__init__.py

```diff
@@ -117,6 +117,15 @@ def __init__(self, fault_identifier, callback, **kwargs):
         self.callback = callback
         self.kwargs = kwargs
 
+    def _repr_first(self):
+        if isinstance(self.id_list, list):
+            return f"<Fault: {self.id_list[0]}>"
```

Comment:
I'm not 100% sold on this. If a secret is formatted into something else, i would like to know that. On the other hand, `id_list` might get very long, and we don't want to clutter the whole screen?

Reply:
It can get very, see reasoning here: https://github.com/bundlewrap/bundlewrap/pull/758/files#diff-58547683697529b0873aa6b387d16f0fefc4e1430e101f8d54198336b62a7b31R305-R312

The usual `__repr__()` does this:

![image](https://github.com/bundlewrap/bundlewrap/assets/82671/4e417bf5-ba16-4907-8602-c978764ea0e6)

While `_repr_first()` does this:

![image](https://github.com/bundlewrap/bundlewrap/assets/82671/3d15b90c-0bab-428f-ac6b-72d721f1c855)

Reply:
… I just had a use case where the *full list* would have been helpful. Need to think about this.

`--unresolved-faults-full` would do the trick, but feels quite clunky. 🫤

Reply:
I suggest merging this as is. `--unresolved-faults-full` is probably very rare. Let’s see how often we actually need that.